demo/lib/TransformationControls/addTableCollapseControl.js

```diff
@@ -1,14 +1,19 @@
 /* eslint-disable require-jsdoc */
 
+const collapseTables = (iframeWindow, iframeDocument) => {
+  event.target.disabled = true
+  if (iframeWindow.pagelib.c1) {
+    // TODO: collapseTables is not exposed in the abstraction layer
```

Comment:
FYI: It's available in the abstraction layer only as part of the `setMulti` call. The apps said they wouldn't need to change the value after the the page is loaded. That's why there is no separate function. Not sure if that will change in the future but likely won't be happening soon.